### DISCUSSION TITLE
Revpop development #revolutionpopuli

### DISCUSSION THREAD

**Discussion by Stamposhi** [community/community#30523] [Other Feature Feedback, Questions, & Ideas]
Is Revpop still being developed?

**Comment by dmitryChernukh** [comment]
I think yes
https://twitter.com/rev_populi?lang=en

**Comment by Stamposhi** [comment]
You think? but no twitter post since March. And github updates are rare and non consistent…

**Comment by jfullstackdev** [comment]
you mean this ? 

https://github.com/Revolution-Populi/revpop-core

you can ask them by raising an issue, although yes, that's one downside of open source, there is no commitment when it comes to support from the developers or maybe there is but you need to pay for it, that's why custom software is still there, companies with even pay large amount of money just to get the support and software they need

**Reply by airtower-luna** [reply]
That's not a downside of open source though, it's a downside of not having a support contract. If an open source developer stops working on something you at least have the option of continuing yourself (or paying someone to), with proprietary software you can't do anything.
